tests/test_cmd.sh

```diff
@@ -34,7 +34,7 @@ fi
 if [[ "$#" -ge 3 ]]; then
   TMP_DIR="$(eval echo "$3")"
```

Comment:
Is this still used in other places or can it be removed?

Reply:
I plan to use it internally.